spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  describe 'GET /show' do
+    it 'returns http success' do
+      get '/users/1/posts/1'
+      expect(response).to have_http_status(:success)
+    end
+  end
+
+  describe 'GET /index' do
+    it 'returns http success' do
+      get '/users/1/posts/'
+      expect(response).to have_http_status(:success)
+    end
+  end
+end
```

Comment:
- [ ] Good job working on your controller specs. However, you seem to only have handled one end of the equation. Kindly include tests that ensures correct view template is rendered. Example : expect(response).to render_template(:show) 
- [ ] Kindly also include tests that ensures the response body includes the correct placeholder text. Example : expect(response.body).to include("Welcome to user index page")